kitchen-sink-example.yaml

```diff
@@ -26,7 +71,44 @@ logger_provider:
     attribute_count_limit: 128
 
 # Configure meter provider.
-meter_provider: {}
+meter_provider:
+  metric_readers:
+    # Add a pull-based metric reader.
+    - type: pull
+      args:
+        exporter:
+          <<: *prometheus-exporter
```

Comment:
#nit: since this is the only use of the prometheus exporter config, how about we include that directly here?

Reply:
Expanded the config and removed the anchor